src/echo/apis/history-api.js

```diff
@@ -99,6 +99,17 @@ class HistoryAPI extends BaseEchoApi {
 		return this.exec('get_relative_contract_history', [contract, stop, limit, start]);
 	}
 
+	/**
+	 * @param {String} address
+	 * @param {Integer_t["uint64"]["__TOutput__"]} stop
```

Comment:
It seems to me that there should be id objects instead of integers in `stop` and `start` arguments. But I could be wrong.